src/test/kotlin/uk/gov/justice/digital/hmpps/prisonercontactregistry/integration/PrisonerContactControllerTest.kt

```diff
@@ -362,8 +362,26 @@ class PrisonerContactControllerTest : IntegrationTestBase() {
     val prisonerId = "A1234AA"
     val visitorIds: List<Long> = listOf(2187526)
     val visitorIdsString = visitorIds.joinToString(",")
-    val fromDate: LocalDate = LocalDate.now()
-    val toDate: LocalDate = LocalDate.now()
+    val fromDate: LocalDate = LocalDate.of(2024, 5, 1)
+    val toDate: LocalDate = LocalDate.of(2024, 5, 10)
+    val uri =
+      "/prisoners/$prisonerId/approved/social/contacts/restrictions/banned/dateRange?visitors=$visitorIdsString&fromDate=$fromDate&toDate=$toDate"
+
+    prisonApiMockServer.stubGetOffenderMultipleContacts(prisonerId)
+
+    webTestClient.get().uri(uri)
+      .headers(setAuthorisation(roles = listOf("ROLE_PRISONER_CONTACT_REGISTRY")))
+      .exchange()
+      .expectStatus().isNotFound
+  }
```

Comment:
should have mention this yesterday...
WE separate our test into     // Given // When // Then see other test.

Reply:
```
  @Test
  fun `when reserve visit slot is successful then CREATED status is returned`() {
    // Given
    val prisonerId = "A123567B"
    val referenceCreated = "aa-bb-cc-dd"
    val createApplicationDto = createCreateApplicationDto(prisonerId)
    val applicationDto = createApplicationDto(reference = referenceCreated)
    visitSchedulerMockServer.stubCreateApplication(applicationDto)

    // When
    val responseSpec = createInitialApplication(webTestClient, createApplicationDto, roleVSIPOrchestrationServiceHttpHeaders)

    // Then
    responseSpec.expectStatus().isCreated
      .expectBody()
      .jsonPath("$.reference").isEqualTo(referenceCreated)
  }
```

Reply:
I've implemented this change, thank you!